Makefile.third

```diff
@@ -57,6 +57,16 @@ $(FRIBIDI_LIB) $(FRIBIDI_DIR): $(THIRDPARTY_DIR)/fribidi/CMakeLists.txt
 	cp -fL $(FRIBIDI_DIR)/lib/$(notdir $(FRIBIDI_LIB)) $@
 	chmod 755 $@
 
+$(LIBUNIBREAK_LIB) $(LIBUNIBREAK_DIR): $(THIRDPARTY_DIR)/libunibreak/CMakeLists.txt
+	install -d $(LIBUNIBREAK_BUILD_DIR)
+	cd $(LIBUNIBREAK_BUILD_DIR) && \
+		$(CMAKE) $(CMAKE_FLAGS) -DCC="$(CC)" -DCFLAGS="$(CFLAGS)"\
+		-DLDFLAGS="$(LDFLAGS)" -DCHOST=$(CHOST) \
+		$(CURDIR)/$(THIRDPARTY_DIR)/libunibreak && \
+		$(CMAKE_MAKE_PROGRAM) $(CMAKE_MAKE_PROGRAM_FLAGS)
+	cp -fL $(LIBUNIBREAK_DIR)/lib/$(notdir $(LIBUNIBREAK_LIB)) $@
+	chmod 755 $@
+
```

Comment:
Time will tell if XCode does funky shit with that one on the DARWIN front, so don't bother for now ;).